.github/actions/deploy-start-action/action.yml

```diff
@@ -51,6 +51,10 @@ runs:
           channel=next
         else
           echo "couldn't resolve release channel based on the pushed tag"
+          echo "for go releases:"
```

Comment:
tbd: @rannn505 migrate all cd-*.yml to work with the <package>/v<X.Y.Z> convention and enforce it via gh rule.